src/PrestaShopBundle/Resources/views/Admin/Improve/Design/positions.html.twig

```diff
@@ -139,7 +139,7 @@
                         {% endif %}
 
                         <div class="module-column-icon">
-                          <img src="{{ asset('/modules/' ~ moduleInstance.name ~ '/logo.png') }}" alt="{{ moduleInstance.displayName| escape }}" />
+                          <img src="{{ asset('../modules/' ~ moduleInstance.name ~ '/logo.png') }}" alt="{{ moduleInstance.displayName| escape }}" />
```

Comment:
sounds like risky with pretty/non pretty urls thing. How about passing modules folder path from the controller?

Reply:
We don't want the folder path but the uri path, but if you're using sub directories like @marionf you will not see images, and we don't have any `base_path` /  `base_uri` for general app (front & back and asset display /admin-dev/)

Reply:
but this way, will the uri path be right in both pretty url and dirty url modes?

Reply:
Yep it will :)